spec/models/order_spec.rb

```diff
@@ -35,6 +34,20 @@
           expect(order.errors.keys).to include(:status)
         end
       end
+
+      # data validations
+      it "validates :status, presence: true, format: { with: VALID_STATUS_REGEX }"
```

Comment:
can you add `# todo` tags to each of these? I usually finish the do/end tags, then put the comment in the body. It won't show up when you run rspec.